v6 analyses page (#60)
EBI-Metagenomics/notebooks: src/docs/glossary.md

Main eukaryotic ribosomal RNA genes used for taxonomic assignments.

## Amplicon

> Refers to an environmental sample where a marker gene has been amplified and sequenced. On the MGnify website, we use the term amplicon when the amplified marker gene is a ribosomal RNA gene. Analysis will yield taxonomic information.

Reviewer: an amplicon isn't a sample... it's a specific bit of sequence that gets amplified (in our case from environmental samples typically) and sequenced. the choice of amplicon or marker gene is then made dependent on the application... maybe something like this?
```suggestion
Refers a marker gene that has been amplified and sequenced. On the MGnify website, we use the term amplicon when the amplified marker gene is a ribosomal RNA gene. Analysis will yield taxonomic information.
```

## Amplicon Sequence Variant (ASV)

> An inferred single DNA sequence recovered from high-throughput analysis of marker genes after removing errors introduced during PCR and sequencing. ASVs can distinguish sequence variation down to a single nucleotide and provide finer resolution than sequence clusters such as [OTUs](#otu).

Reviewer: ```suggestion
An inferred unique DNA sequence recovered from high-throughput analysis of marker genes after removing errors introduced during PCR and sequencing. ASVs can distinguish sequence variation down to a single nucleotide and provide finer resolution than sequence clusters such as [OTUs](#otu).
```
